dhcp6server_test.go

```diff
@@ -0,0 +1,250 @@
+// Copyright 2015 dhcp6 Author. All Rights Reserved.
```

Comment:
Since none of the other files have a header, I think we should omit this one too.

Reply:
In addition, this appears to be in the wrong package.